projects/final-project/solution/danieleFiocca/index.py

```diff
@@ -0,0 +1,237 @@
+print(f"Total breakfast macros: protein {breakfast_protein:.0f}g | carbs {breakfast_carbs:.0f}g | fat {breakfast_fat:.0f}g\n")
+
+print("*" * 60)
+print(f"Snak {snack_percentage:.0f} cal")
+print("*" * 60)
+
+print(f"\nApple: {qty_apple:.0f} g")
+print(f"Walnuts: {qty_walnuts:.0f} g\n")
+snak_protein = (foods["apple"][1] * qty_apple / 100) + (foods["walnuts"][1] * qty_walnuts / 100)
+snak_carbs = (foods["apple"][2] * qty_apple / 100) + (foods["walnuts"][2] * qty_walnuts / 100)
+snak_fat = (foods["apple"][3] * qty_apple / 100) + (foods["walnuts"][3] * qty_walnuts / 100)
+print(f"Total Snak macros: protein {snak_protein:.0f}g | carbs {snak_carbs:.0f}g | fat {snak_fat:.0f}g\n")
+
+print("*" * 60)
+print(f"Lunch {lunch_percentage:.0f} cal")
+print("*" * 60)
+
+print(f"\nRice {qty_rice:.0f}g")
+print(f"Chicken Breast {qty_chicken:.0f}g")
+print(f"Broccoli {qty_broccoli:.0f}g")
+print(f"Olive oil {qty_oil:.0f}g")
+lunch_protein = (foods["rice"][1] * qty_rice / 100) + (foods["chicken breast"][1] * qty_chicken / 100) + (foods["broccoli"][1] * qty_broccoli / 100) + (foods["olive oil"][1] * qty_oil / 100)
+lunch_carbs = (foods["rice"][2] * qty_rice / 100) + (foods["chicken breast"][2] * qty_chicken / 100) + (foods["broccoli"][2] * qty_broccoli / 100) + (foods["olive oil"][2] * qty_oil / 100)
+lunch_fat = (foods["rice"][3] * qty_rice / 100) + (foods["chicken breast"][3] * qty_chicken / 100) + (foods["broccoli"][3] * qty_broccoli / 100) + (foods["olive oil"][3] * qty_oil / 100)
+print(f"Total Lunch macros: protein {lunch_protein:.0f}g | carbs {lunch_carbs:.0f}g | fat {lunch_fat:.0f}g\n")
+
+print("*" * 60)
+print(f"Dinner for {dinner_percentage:.0f} cal")
+print("*" * 60)
+
+print(f"\nSalmon {qty_salmon:.0f}g")
+print(f"Quinoa {qty_quinoa:.0f}g")
+print(f"Spinach {qty_spinach:.0f}g")
+print(f"Oliva oil {qty_oil:.0f}g\n")
+
+dinner_protein = (foods["salmon"][1] * qty_salmon / 100) + (foods["quinoa"][1] * qty_quinoa / 100) + (foods["spinach"][1] * qty_spinach / 100) + (foods["olive oil"][1] * qty_oil / 100)
+dinner_carbs = (foods["salmon"][2] * qty_salmon / 100) + (foods["quinoa"][2] * qty_quinoa / 100) + (foods["spinach"][2] * qty_spinach / 100) + (foods["olive oil"][2] * qty_oil / 100)
+dinner_fat = (foods["salmon"][3] * qty_salmon / 100) + (foods["quinoa"][3] * qty_quinoa / 100) + (foods["spinach"][3] * qty_spinach / 100) + (foods["olive oil"][3] * qty_oil / 100)
+print(f"Total Dinner macros: protein {dinner_protein:.0f}g | carbs {dinner_carbs:.0f}g | fat {dinner_fat:.0f}g\n")
+
+total_protein_day = breakfast_protein + snak_protein + lunch_protein + dinner_protein
+total_carbs_day = breakfast_carbs +snak_carbs + lunch_carbs + dinner_carbs
+total_fat_day = breakfast_fat + snak_fat + lunch_fat + dinner_fat
+
+print("-" * 60)
+print(f"The amount of macronutrients for the day is: ")
+print("-" * 60)
+print(f"\n- Protein: {total_protein_day:.0f}g")
+print(f"- Carbs: {total_carbs_day:.0f}g")
+print(f"- Fat: {total_fat_day:.0f}g\n\n")
```

Comment:
using the `:.0f` notation the number will be truncated before the comma even if the numer is `x.99`... can you find a better solution to handle `float` to round to `int`